demo/src/App.js

```diff
@@ -58,22 +83,37 @@ export default class App extends PureComponent {
       zoom: -5.5,
       target: [30000, 10000, 0]
     };
-    const { sliderValues, colorValues, viewHeight, viewWidth } = this.state;
+    const {
+      sliderValues,
+      colorValues,
+      viewHeight,
+      viewWidth,
+      channelsOn
+    } = this.state;
     const sliders = Object.keys(this.sliders).map(channel => {
       const ChannelSlider = this.sliders[channel];
+      const ChannelCheckbox = this.checkboxes[channel];
       const sliderValue = sliderValues[channel];
       return (
         <div key={`container-${channel}`}>
           <p>{channel}</p>
-          <ChannelSlider
-            value={sliderValue}
-            onChange={(e, v) => this.handleSliderChange(e, v, channel)}
-            valueLabelDisplay="auto"
-            getAriaLabel={() => channel}
-            min={0}
-            max={this.max}
-            orientation="horizontal"
-          />
+          <div style={{ width: '100%', display: 'flex', position: 'relative' }}>
+            <ChannelCheckbox
+              // eslint-disable-next-line no-unused-vars
+              onChange={e => this.toggleChannel(channel)}
```

Comment:
```suggestion
              onChange={() => this.toggleChannel(channel)}
```
and then you don't need the ignore.